package.json

```diff
@@ -88,7 +88,8 @@
     "money": "^0.2.0",
     "morgan": "^1.9.0",
     "multer": "^1.4.2",
-    "navigation-component": "topcoder-platform/navigation-component#develop",
+    "navigation-component": "github:topcoder-platform/navigation-component#develop",
+    "navigation-component-tco": "github:topcoder-platform/navigation-component-tco#new-dev",
```

Comment:
@kkartunov We going to use this component with `new-dev`? Maybe we can create a release from `navigation-component-tco` and use that instead github link?